core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -471,6 +471,11 @@ private void closeObserver(Listener<RespT> observer, Status status, Metadata tra
     observer.onClose(status, trailers);
   }
 
+  @Override
+  public String toString() {
+    return super.toString() + "method=[" + method + "]";
```

Comment:
We are using Guava `MoreObjects.ToStringHelper` instead of harding coding `=[]`

Ditto.